README.md

```diff
@@ -256,23 +258,35 @@ For example:
 <div data-action="click->controller#method">Hello</div>
 ```
 
-If you have multiple actions and/or methods on the same element, pass them all as an
-associative array in the first argument:
+If you have multiple actions and/or methods on the same element, you can chain them as there's also a
+`stimulus_action` filter:
 
 ```twig
-<div {{ stimulus_action({
- 'controller': 'method',
- 'other-controller': ['method', {'resize@window': 'onWindowResize'}]
-}) }}>
+<div {{ stimulus_action('controller', 'method')|stimulus_action('other-controller', 'test') }}>
     Hello
 </div>
 
 <!-- would render -->
-<div data-action="controller#method other-controller#method resize@window->other-controller#onWindowResize">
+<div data-action="controller#method other-controller#test">
     Hello
 </div>
 ```
 
+You can also pass [parameters](https://stimulus.hotwired.dev/reference/actions#action-parameters) to actions:
+
+```twig
+<div {{ stimulus_action('hello-controller', 'method', 'click', { 'count': 3 }) }}>Hello</div>
+
+<!-- would render -->
+<div data-action="click->hello-controller#method" data-hello-controller-count-param="3">Hello</div>
+```
+
+You can also retrieve the generated attributes as an array, which can be helpful e.g. for forms:
+
+```twig
+{{ form_row(form.password, { attr: stimulus_action('hello-controller', 'checkPasswordStrength').toArray() }) }}
+```
```

Comment:
Let's move the `.toArray()` example above the "action parameters" example.

Reply:
Done :)